blocks/init/src/Blocks/custom/tabs/assets/tabber.js

```diff
@@ -4,10 +4,11 @@ export class Tabber {
         this.tabsBlockSelector = options.tabsBlockSelector;
         this.tabsBlocks = document.querySelectorAll(this.tabsBlockSelector);
         this.blockClass = options.blockClass;
-        this.blockJsClass = options.blockJsClass
-        this.tabJsClass = options.innerBlockJsClass;
+        this.blockJsClass = options.blockJsClass;
+        this.tabClass = options.innerBlockClass;
+        this.tabJsClass = `js-${this.tabClass}`;
 
-        this.IS_VISIBLE_CLASS = 'is-visible-tab';
+        this.IS_VISIBLE_CLASS = `${this.tabClass}--is-visible`;
```

Comment:
`.is-visible` shouldn't be used as a modifier. The action modifiers (`is-visible`, `is-active`, `hidden`, etc.) can be set as is, not following BEM. BEM modifiers are used only to denote specific style change, not action.

Reply:
I can change that back, [but your comment is the reason I changed it to be a modifier](https://github.com/infinum/eightshift-frontend-libs/pull/412#discussion_r681548993). I'm still kinda new to BEM naming 😅